ansible/inventory/teleop/group_vars/simulation.yml

```diff
@@ -5,6 +5,7 @@ reinstall: false # Flag to know if the docker container should be fully reinstal
 nvidia_docker: true # Define if nvidia-container-toolkit is used (defaul is false)
 glove: "shadow_glove" # Determines which glove is installed (haptx or shadow_glove)
 container_name: "teleop_{{ glove }}" # Name of the docker container
+bimanual: true # when doing sim we should always have it bimanual.
```

Comment:
out of curiosity, why?

Reply:
We always create bimanual icons for teleop in simulation, this just makes sure these new folders get created :)